### ISSUE TITLE
Uncaught TypeError: Cannot read property 'clipboardData' of undefined when pasting

### ISSUE THREAD

**Issue by ruionwriting** [angular-ui/ui-select#848]
When pasting anything - all modes - the console throws the presented error.

**Comment by bobs619** [comment]
I ended up changing the spot where it errors to this:

```javascript
var t = e.originalEvent.clipboardData ? e.originalEvent.clipboardData.getData("text/plain") : window.clipboardData.getData("text");
```

I tested it on IE 11 and Chrome.  Works!

**Comment by rysilva** [comment]
The issue is that originalEvent is jquery.  Since jquery is supposedly not required for ui-select, this reference should be removed completely for the fix.

**Comment by sdir456** [comment]
+1

**Comment by shali3** [comment]
+1 Please fix

**Comment by cfey** [comment]
hey guys, do you have any progress on this?

**Comment by shali3** [comment]
I created a pull request #1391 to fix this yesterday.

**Comment by cfey** [comment]
ah, thanks
+1

**Comment by erikdubbelboer** [comment]
There are many pull requests for this such as #1276 , But the project isn't maintained anymore so nobody merges them.

**Comment by cfey** [comment]
Is there a way to keep this going?
Would be sad if this project gets abandoned :confused:

**Comment by shali3** [comment]
I opened #1398 
You are welcome to comment there

**Comment by aaronroberson** [comment]
Please confirm fix in v0.14.7 and close this issue.

**Comment by wesleycho** [comment]
Closing this as fixed - feel free to open a new issue if it is still a problem.
